### ISSUE TITLE
Textbook Style Transfer Loss

### ISSUE THREAD

**Issue by loparcog** [McMasterAI/StyleGAN#26]
# What is it

The style transfer model given in the textbook example in our teams chat uses a triple-section loss function, analyzing style, content, and "realness" of the generated image. This would be more complex than the current loss function we are using, and may produce better results.

# What does it change

This would change training, specifically the loss functions, and *maybe* the discriminator but I don't think it needs to

# Goals to see

This would hopefully provide better feedback to the generator, allowing it to learn and grow faster/better than the base model
